client/src/__tests__/PrescriptionCreation/index.spec.tsx

```diff
@@ -217,4 +219,70 @@ describe('PrescriptionCreation', () => {
 
     expect(screen.getByText(/Soumettre/i).closest('button')).toBeEnabled();
   }, 10000);
+
+  test('Prescriptions: Create prescription button', async () => {
```

Comment:
the `Prescriptions:` is not required since, when running, it creates a "tree" or a "path" so you'll already have it from the parent `describe` (they can be nested too)